bin/flutter.bat

```diff
@@ -86,11 +86,11 @@ GOTO :after_subroutine
   IF NOT EXIST "%stamp_path%" GOTO do_snapshot
   SET /P stamp_value=<"%stamp_path%"
   IF !stamp_value! NEQ !revision! GOTO do_snapshot
-  REM Compare "last modified" timestamps
+  REM Compare created timestamps
```

Comment:
This change changes the semantics though, no? Previously, if I'd add a dependency to pubspec.yaml during dev it would rebuild the snapshot because "last modified" changed. Now it doesn't do that anymore.

I could get behind the argument that we don't need that convenience and require you to delete the `flutter_tools.stamp` whenever you change the pubspec (like we require you to do for any other change to flutter_tools as well). However, in that case what's the point of comparing the creation time of yaml and lock? Would we loose anything by removing the check all-together?

Reply:
That's a good point. I think you're right and this comparison is redundant now anyway. Removed it entirely.